cmd/kube-proxy/app/validation.go

```diff
@@ -90,6 +90,22 @@ func validateKubeProxyIPTablesConfiguration(config componentconfig.KubeProxyIPTa
 	return allErrs
 }
 
+func validateKubeProxyIPVSConfiguration(config componentconfig.KubeProxyIPVSConfiguration, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if config.SyncPeriod.Duration <= 0 {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("SyncPeriod"), config.SyncPeriod, "must be greater than 0"))
+	}
+
+	if config.MinSyncPeriod.Duration < 0 {
```

Comment:
Do we need to check if MinSyncPeriod > SyncPeriod ?

Reply:
That's a nice finding! 

I think we can check `MinSyncPeriod > SyncPeriod` in validation although currently it's already checked in proxier.go. Let's change it to see if anyone object to it :)

Reply:
>although currently it's already checked in proxier.go

IMO, it's better validating earlier than later :)